themes/contrib/emulsify/templates/fields/field--block-content--field-cta-block--homepage-cta.html.twig

```diff
@@ -0,0 +1,5 @@
+<div class="grid grid--{{ items | length }}">
```

Comment:
@johnhwhite This is super clever! `grid--{{ items | length }}` I'm definitely going to use that!

However, all markup, should be done in Pattern Lab, so that there's only one canonical source of markup.

The Drupal templates should just have `extend` and `include` using `with` to wire up the Drupal content.